jax/experimental/stax.py

```diff
@@ -30,67 +30,27 @@
 
 from jax import lax
 from jax import random
-from jax.scipy.special import logsumexp, expit
 import jax.numpy as np
 
+from jax.nn import *
```

Comment:
How hard would it be to make this import explicit? I think we have avoided `from stuff import *` so far in the code (except in `__init__.py` files) and it's a bit of an anti-pattern.

Could also `from jax.nn.initializers import glorot_normal` etc to make some lines shorter below.